docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,68 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
 
+## Rigid Waist Shift & LSA Knobs
+
+Some information about the rigid waist shift can be found in [this gallery page][rws_gallery]{target=_blank} from Felix's package's documentation.
+Some details about the creation of these knobs and their addition in LSA can be found in this [LHC OMC logbook entry](https://be-op-logbook.web.cern.ch/elogbook-server/GET/showEventInLogbook/3545713){target=_blank} and the ones that follow.
+
+### Knob Setting Convention
+
+Since the waist can be shifted in two directions - left and right of the IP - two knobs were generated per IP, for a unit setting of $1$ and -$1$.
+This unit setting is an arbitrary definition, where a value of $1$ corresponds to a $0.5$% change of the triplet powering.
```

Comment:
I would rather write 
> This unit setting is an arbitrary definition, where a value of $\pm1$ corresponds to a waist shift of ~43-44cm in a given direction, corresponding to a  $\pm 0.5$% change of the triplet powering.

as the shift distance is probably more important

Reply:
This is indeed much better, thanks!

Reply:
Adressed in [14abe46](https://github.com/pylhc/pylhc.github.io/pull/74/commits/14abe462aac917a757b1b6b3d51852e686029608)